docker/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+set -ex
+
+TOKEN=${TOKEN} npm start
```

Comment:
Does this allow for no token? how are we passing this in?

Reply:
Yup, it allows for no token, but it should be passed in as an environment variable when you run docker run: """\$JENKINS_HOME/scripts/run_testing_container.py --cluster \${ECS_CLUSTER_NAME} --product ${COMPONENT_RENDERER_SERVICE_NAME} --product-version ${COMPONENT_RENDERER_IMG_VERSION} --proxy-port ${COMPONENT_RENDERER_PORT} --env-vars TOKEN=\${PLATUI_SERVICE_ACCOUNT_GITHUB_TOKEN} --subnet-ids \${CONTAINER_SUBNET_IDS} --sg-ids \$${COMPONENT_RENDERER_SG_IDS_VAR_NAME} --build-tag "\$BUILD_URL\""""

Reply:
This is done in the build job - token should be stored by B&D in Build Jenkins - a blank token doesn't seem to cause the service to fail :)